packages/lib/src/select/Select.tsx

```diff
@@ -520,6 +523,8 @@ const ClearOptionsAction = styled.button`
   color: ${(props) =>
     props.disabled ? props.theme.disabledColor : props.theme.enabledSelectionIndicatorActionIconColor};
   font-size: 16px;
+  width: 100%;
+  height: 100%;
```

Comment:
`height: 100%;` is not needed, by default, a grid container always occupies the full height.

Reply:
Correct.